cristin-commons/src/main/java/no/unit/nva/utils/DateInfo.java

```diff
@@ -1,4 +1,4 @@
-package no.unit.nva.cristin.projects.model.nva;
+package no.unit.nva.utils;
```

Comment:
This is a model class, not a util class. I think a better place would be to put DateInfo in model package, where Organization.java is. The module seems right. Things we need for more than one module should be in commons module, as you have done here